### ISSUE TITLE
Oculus Touch Controller Mappings not working in Unreal 4.25

### ISSUE THREAD

**Issue by PapeCoding** [JackD83/ALVR#239]
# Please follow the general troubleshooting steps first:

- [x] I ran "add_firewall_rules.bat" as administrator and checked that the ports are open in the firewall
- [x] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/JackD83/ALVR/wiki)

### Bug reports:

The Oculus Touch Controller Mappings are not working in Unreal 4.25. The Trigger is recognized, as well as the A, B, X, Y buttons, but the thumb sticks are not recognized at all. If needed, a sample Unreal application can be provided to test this.

#### Environment:

i7-7700, GTX1060, Generic Realtek Stuff
ALVR v12.0 beta 2 + SteamVR 1.13.10
The latest Windows 10 Updates are installed

#### Feature:

A tutorial on how the controller mappings work, would be appreciated! Are they hard-coded? Can they be remapped easily?

**Comment by PapeCoding** [comment]
After short testing with Unreal 4.22, the `MotionController (R) Thumbstick X` worked. Since Unreal changed its input system, this is not working anymore.

**Comment by zarik5** [comment]
ALVR input mappings are hardcoded, but you can change them within SteamVR.

**Comment by PapeCoding** [comment]
Ok, this is definitely not a fault from this software. The SteamVR plugin inside Unreal is generating these bindings in a weird fashion.

For anyone else having this issue, look [here](https://forums.unrealengine.com/development-discussion/vr-ar-development/1768997-trackpad-axis-movement-input-logic-4-25?p=1774714#post1774714) or [here](https://github.com/ValveSoftware/steamvr_unreal_plugin/issues/142).

In short:
The plugin maps the scalar axis that Unreal uses to the vector axis that SteamVR uses. But it does this only if you request both parts of the vector. You either call your Axis `<something>_X` and a second one `<something>_Y` or you include both axis in both mappings and scale one of them to 0.

Thank you @zarik5, you brought me to the solution!
